src/peer/peer_manager.cpp

```diff
@@ -336,14 +337,17 @@ void PeerManager::CheckTimeout() {
     }
 }
 
+bool PeerManager::InitialSyncCompleted() const {
+    return DAG->GetMilestoneHead()->cblock->GetTime() >= (time(nullptr) - kSyncTimeThreshold);
```

Comment:
I suggest use the variable `initial_sync_`

Reply:
Changed